server/package.json

```diff
@@ -3,19 +3,21 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
-    "scripts": {
-      "test": "echo \"Error: no test specified\" && exit 1",
-      "dev": "nodemon index.js",
-      "prod": "SET NODE_ENV=production&& nodemon index.js"
-    },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "SET NODE_ENV=development&&nodemon index.js",
```

Comment:
did it raise any issues?, because it will be useful when we separate our project into different environment.

Reply:
yes this will only work in windows not in other OS. and on windows machine itself it was not working for the person who message in group i checked in machine i could not figure why it was not working so i changed the logic.

Reply:
ok bro, i will revert and share your's changed logics